lib/ex_unit/lib/ex_unit/assertions.ex

```diff
@@ -35,6 +35,19 @@ defmodule ExUnit.Assertions do
   common cases such as checking a floating point number or handling exceptions.
   """
 
+  @doc false
+  defmacro raise_assert(opts) do
```

Comment:
I don't like this because we are introducing a new API. What if we allow flunk(%ExUnit.AssertionError{}) that will call :erlang.error and we catch it?